src/components/EarningsTable/EarningsTable.js

```diff
@@ -139,12 +139,16 @@ function EarningsTable() {
     const filtersToSubmit = { ...filter };
     // filter out keys we don't want to submit
     Object.keys(filtersToSubmit).forEach((k) => {
-      if (
-        filtersToSubmit[k] === 'all' ||
-        filtersToSubmit[k] === '' ||
-        k === 'organization_id'
-      ) {
-        delete filtersToSubmit[k];
+      if (k == 'grower') {
```

Comment:
Let's compare it as `===`. wdyt?

Reply:
changed the comparison from "==" to "==="